development/docker-compose/docker-compose.yaml

```diff
@@ -244,32 +248,29 @@ services:
     links:
       - registry
 
-  product-catalog-task:
-    image: repo.backbase.com/backbase-stream-images/product-catalog-task:${STREAM_VERSION}
+  bootstrap-job:
+    image: bootstrap-job:${BOOTSTRAP_JOB_VERSION}
     profiles:
       - bootstrap
     environment:
       <<: *common-variables
-      eureka.client.register-with-eureka: false
-      spring.config.additional-location: file:/tmp/ingestion.yml
-    volumes:
-      - ../config/stream/ingestion.yml:/tmp/ingestion.yml
-    depends_on:
-      token-converter:
-        condition: service_healthy
-      arrangement-manager:
-        condition: service_healthy
-
-  legal-entity-bootstrap-task:
-    image: repo.backbase.com/backbase-stream-images/legal-entity-bootstrap-task:${STREAM_VERSION}
-    profiles:
-      - bootstrap
-    environment:
-      <<: *common-variables
-      eureka.client.register-with-eureka: false
-      spring.config.additional-location: file:/tmp/ingestion.yml
-    volumes:
-      - ../config/stream/ingestion.yml:/tmp/ingestion.yml
+      spring.autoconfigure.exclude: org.springframework.boot.autoconfigure.security.oauth2.client.reactive.ReactiveOAuth2ClientAutoConfiguration
+      backbase.communication.http.discoverable-access-token-service: true
+      backbase.stream.limits.worker.enabled: false
+      backbase.bootstrap.data.active-data-sets: local-backend-setup
```

Comment:
where are these files? I assumed they would be included in this repo just like we used to have for the previous jobs

Reply:
The data will be downloaded after the script is executed. it can be found here: `development/images/bootstrap/target/bootstrap-job/data/src/main/resources/local-backend-setup`. The instruction on how to manipulate it, is also elaborated [here](https://github.com/Backbase/local-backend-setup/blob/BEFOUND-1640-replace-stream-with-bootstrap-job/development/docker-compose/readme.md?plain=1#L166).

Reply:
according to our last change, it is explained [here](https://github.com/Backbase/local-backend-setup/pull/28/files#diff-dad0be724a0907558a154722be2b9961e0a8594fae90befc907d1791f3af22c9).